source/extensions/filters/http/jwt_authn/authenticator.cc

```diff
@@ -182,7 +187,7 @@ void AuthenticatorImpl::startVerify() {
     if (!fetcher_) {
       fetcher_ = create_jwks_fetcher_cb_(cm_);
     }
-    fetcher_->fetch(jwks_data_->getJwtProvider().remote_jwks().http_uri(), *this);
+    fetcher_->fetch(jwks_data_->getJwtProvider().remote_jwks().http_uri(), *parent_span_, *this);
```

Comment:
is it possible that this parent_span_ is null?  If so, need to handle it.

Reply:
It should not be possible, since `verify(...)` is the only public entry-point into the `authenticator.cc`, and the `parent_span_` is set to to the passed in public span directly in that function.

Just in case, I will initialize the pointer to point to the static `NullSpan`. That way if it ever is not initialized, it will not cause a segfault.